bugbug/repository.py

```diff
@@ -487,10 +487,9 @@ def update_experiences(experience_type, day, items):
                 or commit_type == "backout"
                 and commit.backedoutby
             ):
+                exps = experiences[experience_type][commit_type]
```

Comment:
before: experiences was accessed only when items is not empty.
now: experiences is accessed whatever items are that could lead to an error when items is empty.